packages/nimble-components/src/select/specs/built-in-labels.md

```diff
@@ -39,16 +39,15 @@ Default Slot:
 Template updates:
 
 -   A new `label` element will be added at the root of the control template, which will reflect the default slotted content (minus the list options which are already handled elsewhere)
--   A new container `div` will be added at the same level, to contain the control part (`div[part="control"]`)
--   Update element host `display` mode so the label is laid out correctly. (Formerly `inline-flex`, which will move to the container `div`. Prototyped as `inline-block`, but `inline-flex` / `inline-grid` is probably more in line with what we want for our controls layouts going forward.)
-
-Currently our shared dropdown styling (`patterns/dropdown/styles.ts`) applies focus/error-visible styling directly on `::host` - most of those styles will instead target the container `div`.
+-   Update shared dropdown styling as needed for label layout (`flex-direction` to `column`, move fixed height from `::host` down to the `div[part="control"]`)
```

Comment:
After further implementation / style tweaks, I found that I didn't need to add a new container div after all, so I simplified the implementation and HLD here. There's also no changes to the rich text mention listbox template, with the updated approach.

Initial issues I hit when prototyping were (1) to get the `div.control` to still size correctly, (2) focus/ focus-visible styling, including getting the underlines to still center, but those issues are fixed now.